src/content/docs/alerts-applied-intelligence/overview.mdx

```diff
@@ -24,79 +24,94 @@ redirects:
+Alerts and Applied intelligence were created for busy DevOps teams: it reduces noise and makes sure you never miss an issue. 
 
-Already have an account? [Login](https://one.newrelic.com).
+## How it all works 
 
-Alerts lets you define the conditions in your system you want to be notified about, letting you focus on the things you care about most.
+### Detect and diagnose potential problems in your system [#detect-intro]
 
-Applied intelligence is a machine-learning engine that reduces alert noise, correlates incidents, automatically detects anomalies, and provides root cause analysis. Not only that, it enriches your notifications with metadata to give you greater context of what's happening when you are notified.
+New Relic never stops observing your telemetry data and we created alerts so your team can decide when you want to be notified and how.  Applied intelligence takes alerting one step further and monitors for any deviations from what's expected and notifies your team instantly. You can decide what level of automation works best for each situation. Our automatic anomaly detection is easy to use and implement, while our more customizable detection options offer your team the ability to create alerts that are just right for you. 
+
+Learn how to configure anomalies [in our anomaly docs](https://docs.newrelic.com/docs/alerts-applied-intelligence/applied-intelligence/incident-workflows/incident-workflows).
+
+### Correlate relevant issues to reduce noise [#correlation-intro]
+
+After you've detected any irregular behavior in your system, applied intelligence's correlation logic reviews all incoming events and compares them based on signals like time, context, or source. If multiple issues are discovered to be related, instead of creating many, redundant alerts, applied intelligence's correlation logic will group these issues together to create a single, relevant alert. This saves your team time and energy and allows you to get to the heart of the problem faster. 
+
+Our correlation logic is called decisions. Learn how to set up decisions, follow our step by step guide by [in our decisions docs](https://docs.newrelic.com/docs/alerts-applied-intelligence/applied-intelligence/incident-workflows/incident-workflows).
+
+### Notify the right people at the right time [#workflow-intro]
+
+Once you've used our anomaly detection to identify irregular behavior, it's important that the right people on your team are notified of any potential problems. New Relic created workflows so you can control when and where you want to receive notifications about issues. Workflows allow you to funnel the right information to the relevant person or team. Workflows also give you the opportunity to enrich your notifications with additional New Relic data.
+
+To learn how to set up workflows, follow our step by step guide by [in our workflows docs](https://docs.newrelic.com/docs/alerts-applied-intelligence/applied-intelligence/incident-workflows/incident-workflows).
```

Comment:
just a note to review links and make them relative links

Reply:
done!